webapp/src/properties/date/property.tsx

```diff
@@ -65,14 +62,11 @@ export default class DateProperty extends PropertyType {
 
     getDateTo = (value: string | string[] | undefined) => {
         const dateProperty = createDatePropertyFromString(value as string)
-        if (!dateProperty.from) {
+        if (!dateProperty.to) {
             return undefined
         }
-        const dateFrom = dateProperty.from ? new Date(dateProperty.from + (dateProperty.includeTime ? 0 : timeZoneOffset(dateProperty.from))) : new Date()
-        dateFrom.setHours(0, 0, 0, 0)
-
-        const dateToNumber = dateProperty.to ? dateProperty.to + (dateProperty.includeTime ? 0 : timeZoneOffset(dateProperty.to)) : dateFrom.getTime()
-        const dateTo = new Date(dateToNumber + oneDay) // Add one day.
+        const dateToNumber = dateProperty.to + (dateProperty.includeTime ? 0 : timeZoneOffset(dateProperty.to))
+        const dateTo = new Date(dateToNumber)
         dateTo.setHours(0, 0, 0, 0)
```

Comment:
Moved incrementing a day to FullCalendar, since it is for that functionality. Also, don't default to `from`. Return undefined, if not a range.